python/datasources/acs_health_insurance.py

```diff
@@ -235,21 +235,21 @@ def upload_to_gcs(self, bucket):
 
         female_state_params = format_params(
             HEALTH_INSURANCE_BY_SEX_GROUPS_PREFIX, HEALTH_INSURANCE_BY_SEX_FEMALE_SUFFIXES)
-        file_diff = file_diff and url_file_to_gcs.url_file_to_gcs(
+        file_diff = url_file_to_gcs.url_file_to_gcs(
             self.base_url, female_state_params, bucket,
-            self.get_filename(Sex.FEMALE, None, False))
+            self.get_filename(Sex.FEMALE, None, False)) or file_diff
```

Comment:
Could you keep the structure of file_diff = file_diff or url_file_to_gcs... ? IMO it helps with readability

Reply:
No.  This short circuits the code and if one file diff is different will not upload any of the other datasets to GCS